server/models/UserModel/index.js

```diff
@@ -29,7 +29,7 @@ const UserModel = {
     password: '',
     ldap: false,
     force_reset_pw: false,
-    realms: config.app.realms,
+    realms: config.app.realms || [],
```

Comment:
this `config` file that is imported is in the gitignore, which is why the test is failing. Lookin at the config file the `realms` value is `['']`. Maybe we should explicitly set that or an empty array instead of reading from this file?